portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,53 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+private ArrayList<String> names;
+
+  @Override
+  public void init() {
+      names = new ArrayList<String>();
+      names.add("RMP");
+      names.add("Love Me More");
+      names.add("6 Kiss");
+      names.add("Abandoned");
+      names.add("Real Feel");
+      names.add("Chosen");
+      names.add("Can you Rap Like Me -Pt.2");
```

Comment:
Lol are these movies? Would you recommend any?

Reply:
You probably haven't got to it, but eventually, we hope to retrieve data/comments from the DataStore (database)

Reply:
No haha but I do have a movie watch list I've been going through. I'll send you my list and breakdowns. You will hate it:)
Yeah deleted this part and most of the above things I hadn't gotten to the optimized parts.